tests/functional/oauth_webchannel.js

```diff
@@ -39,13 +39,13 @@ define([
    * finish OAuth flows
    */
 
-  function testIsBrowserNotifiedOfLogin(context, shouldCloseTab, wantKeys) {
+  function testIsBrowserNotifiedOfLogin(context) {
```

Comment:
Yeah, I agree with @vladikoff, let's pass in an `options` object here and ditch the state on `context. The `options` object can contain two fields:
* shouldCloseWindow
* wantsKeys